README.md

```diff
@@ -27,6 +27,11 @@ mkdir build && cd build
 cmake --build . --config Release
 ```
 
+4. Installing the library
```

Comment:
This I don't like. We should have an instruction which installs the package in a directory owned by a local user, otherwise we get a following problem:

```
grzanka@grzankax1:~/workspace/gi-yagit/build$ cmake --install .
-- Install configuration: "Release"
-- Up-to-date: /usr/local/include
CMake Error at cmake_install.cmake:51 (file):
  file INSTALL cannot set permissions on "/usr/local/include": Operation not
  permitted.

```